### PR TITLE
Address test warnings

### PR DESCRIPTION
**Pull request by asdacap**
- Many warnings on every pr for tests. 
- This addresses them.
- But there are still warnings that is outside tests.
- Not touching them right now.

## Changes:
- Addresses warnings.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update
- [X] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe): 

## Testing
**Requires testing**

- [ ] Yes
- [X] No

**In case you checked yes, did you write tests??**

- [ ] Yes
- [X] No

**Comments about testing , should you have some** (optional)

## Further comments (optional)

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by asdacap** [comment]
Well... that did not work.

**Review by MarekM25 — APPROVED** [review]
Not sure if you fixed the whole project, but if yes you can add TreatWarningAsErrors in .csproj.

File: src/Nethermind/Nethermind.Core.Test/AddressTests.cs [hunk 1]
```diff
@@ -89,7 +89,9 @@ public void Equals_operator_works()
             Assert.False(addressA == addressB);
             Assert.False(addressA is null);
             Assert.False(null == addressA);
+#pragma warning disable CS8417
             Assert.True((Address?)null is null);
```
[the remark this answers is not in the document]
Comment (asdacap) [reply src/Nethermind/Nethermind.Core.Test/AddressTests.cs:93]: I thought that would work. Seems to work for intellij warning.

[the remark this answers is not in the document]
Comment (asdacap) [reply src/Nethermind/Nethermind.Core.Test/AddressTests.cs:93]: Aha! it work now. Seems to be different code.

**Pull request merged**